packages/dmn-js-decision-table/src/features/decision-rules/index.js

```diff
@@ -4,7 +4,8 @@ import DecisionRules from './DecisionRules';
 
 export default {
   __depends__: [
-    ExpressionLanguagesModule
+    ExpressionLanguagesModule,
+    'translate'
```

Comment:
Does this do anything? We must depend on a module. Depending on a `string` does not yield any meaningful result.